MANUAL.md

```diff
@@ -0,0 +1,647 @@
+
+### Access detailed incident view
+* Alternative 1:
+  1. Click on an incident row in the _Incidents_ table.
+  2. Detailed incident will appear in a pop-up window.
+  
+     <img src="public/screenshots/manual/Screenshot 2022-09-09 at 09.40.01.png"/>
+    
+* Alternative 2:
+  1. Click on one of the icons under _Actions column_ in the _Incidents_ table.
+  
+     <img src="public/screenshots/manual/Screenshot 2022-09-09 at 08.21.36 copy.png"/>
+     
+  2. App will redirect you to the incident's page.
+  
+     <img src="public/screenshots/manual/Screenshot 2022-09-09 at 09.42.33.png"/>
+
+### Work with incidents table
+
+#### Change how many rows are shown per incidents table page
+1. Scroll down to the bottom of the _Incidents_ table.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 09.57.34.png"/>
+   
+2. Click on the _Rows per page_ drop-down.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 09.57.34 copy.png"/>
+   
+3. Select whether you want 10/25/50/100 incidents per page displayed.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 09.58.16.png"/>
+
+#### Navigate incidents table
+1. Scroll down to the bottom of the _Incidents_ table.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 09.57.34.png"/>
+   
+2. Click on the _right arrow icon_ if you want to go to the next table page.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 10.05.05 copy.png"/>
+   
+3. Click on the _left arrow icon_ if you want to go to the previous table page.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 10.05.05.png"/>
+
+
+#### Change how often incidents table gets refreshed
+1. Click on the _gears icon_ to the right below the header.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-09 at 10.08.23.png"/>
+   
+2. Select refresh method in the _Auto Update selector_:
+
+   * If you want **no automatic table updates**, press `NEVER` in the _Auto Update selector_. Note that you will have to refresh the page yourself if you want the table to get updated.
+   
+     <img src="public/screenshots/manual/Screenshot 2022-09-09 at 10.08.45.png"/>
+     
+   * If you want the table to update **in realtime**, press `REALTIME` in the _Auto Update selector_.
+   
+     <img src="public/screenshots/manual/Screenshot 2022-09-09 at 10.08.45 copy.png"/>
+     
+   * If you want the table to get updated **every couple of seconds**, press `INTERVAL` in the _Auto Update selector_.
+   
+     <img src="public/screenshots/manual/Screenshot 2022-09-09 at 10.08.45 copy 2.png"/>
+      
+     * You can see the value of the current refresh interval below the _Incidents table_. The refresh interval is displayed **in seconds**.
+
+       <img src="public/screenshots/manual/Screenshot 2022-09-22 at 08.16.17.png"/>
+     
+     * You can change the value of a refresh interval in `/src/config.tsx`. The refresh interval is stored **in seconds**.
```

Comment:
````suggestion
     * You can change the value the refresh interval in `/src/config.tsx`. The refresh interval is stored **in seconds**.
````

Reply:
But there is only one interval if I understand it correctly

Reply:
"value of" doesn't mean the same as "value".

Reply:
Considering the previous sentence the best solution is probably "You can change the refresh interval value in `/src/config.tsx`." Also, I assume the refresh value is *whole seconds*, not a float?